runtimes/peregrine/src/xcm_config.rs

```diff
@@ -92,7 +89,7 @@ impl xcm_executor::Config for XcmConfig {
 	// How weight is transformed into fees. The fees are not taken out of the
 	// Balances pallet here. Balances is only used if fees are dropped without being
 	// used. In that case they are put into the treasury.
-	type Trader = UsingComponents<WeightToFee<Runtime>, RelayLocation, AccountId, Balances, Treasury>;
+	type Trader = UsingComponents<WeightToFee<Runtime>, HereLocation, AccountId, Balances, Treasury>;
```

Comment:
We allow unpaid execution from the relay chain. Plus, here we were saying that relay chain funds could be used to pay for the fees, when nothing else was allowed. I think keeping it local for now is the most logical choice. We will open up once we open up to other chains.